### PR TITLE
Contact section

### PR DESCRIPTION
**Pull request by VicPeralta**
## Contact section

Add contact section information

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -295,5 +295,20 @@ <h3 class="languages-title">Languages</h3>
...
+      <h3>Contact me</h3>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form action="https://formspree.io/f/mwkypgbn" method="post">
+        <input type="text" name="full-name" required  maxlength="30" placeholder="Full name" class="input-field-contact"><br>
+        <input type="email" name="email" required placeholder="Enter email" class="input-field-contact"><br>
+        <textarea name="comment" id="comment" cols="30" rows="10" placeholder="Write your message here" required></textarea><br>
```
Comment (lily-coder) [inline index.html]: - [x] [REQUIRED] As stated in the project requirements: Please ensure that the message field accepts up to 500 chars. You can do this by adding the `maxlength="500"` attribute

**Review by lily-coder — CHANGES_REQUESTED** [review]
Hi @VicPeralta and @Have-Samuel , 

Good job so far!🎉🎉🎉

To Highlight! 🟢
✔️ Good job with the linter checks
✔️ Great effort in parsing the Figma template.

There are some issues that you still need to work on to go to the next project but you are almost there!

https://user-images.githubusercontent.com/63964646/149004865-8fef09b9-a7cf-4a54-bf04-1744dcf74af0.mp4


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] You can also consider writing your commit messages in the imperative tense instead of the past tense as it is more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -331,3 +340,85 @@ body {
...
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: left;
+}
```
Comment (medaminedev66) [inline styles.css]: [OPTIONAL]
- It would be better to add some paddings to your form inputs to improve your form design.

**Review by medaminedev66 — COMMENTED** [review]
# Hi @VicPeralta and @Have-Samuel , 

### Status: Approved ✔️

:+1: You have implemented all the suggestions. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-  Kindly add a link to the live demo of your app to the readme file.

**You may close this Pull Request and merge the changes**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Amine Snahi_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**